src/android/io/branch/BranchSDK.java

```diff
@@ -259,6 +259,20 @@ private void initSession(CallbackContext callbackContext)
     }
 
     /**
+     * <p>Initialises a session with the Branch API, without a callback or {@link Activity}.</p>
+     */
+    private void initSession()
```

Comment:
can you just put this function in the one above and check `if (!callbackContext)`